pkg/shipper/gcp.go

```diff
@@ -49,9 +56,19 @@ func newGCSRemoteMetrics(r prometheus.Registerer) *gcsRemoteMetrics {
 		Name: "thanos_gcs_remote_upload_failures_total",
 		Help: "Total number of failed object uploads",
 	})
+	m.gcsOperations = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "thanos_gcs_operations_total",
+		Help: "Number of Google Storage operations.",
+	}, []string{"type"})
 
 	if r != nil {
-		prometheus.MustRegister()
```

Comment:
Oops :D

Reply:
yup - this is a proof that if the function designed to do something with given input as `...type` is invoked without anything (which does not make sense), it should return error (: or panic in case of `prometheus.MustRegister()`